frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/simple-create.ts

```diff
@@ -143,7 +143,7 @@ export const prepareVM = async (
     let rootDataVolume;
     let isCDRom: boolean;
 
-    if (customSource?.dataSource?.value) {
+    if (customSource?.dataSource?.value && !sourceStatus) {
```

Comment:
This bug is about a label not showing, why do you need to touch the create proccess ?

/hold
touching create is a bit dangerous, please explain